docs/pages/app-signing/apple-team-permissions.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Apple Developer Program roles and permissions for EAS Build
+sidebar_title: Apple Developer Program roles and permissions
+description: Learn about the Apple Developer account membership requirements for creating an EAS Build.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+When creating a build for and iOS device on EAS, an Apple Developer account is required. This account has necessary permissions to submit the app for review, manage app's distribution, and generate required [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles.
+
+If the Apple Developer account is an individual account, only the account owner can generate the app signing credentials. This guide provides steps that the account owner has to follow to ensure app signing credentials are generated. It also provides steps for the team developer to create an EAS Build.
+
+> See [Apple's documentation on Program Roles](https://developer.apple.com/support/roles/) for details on the different roles and their permissions based on the type of Developer account.
+
+## Steps for Apple Developer account owner
+
+The owner of the individual Apple Developer account needs to generate a distribution certificate (if submitting an app to the App Store), provisioning profile, or a push key (if submitting an app to use a push notification service).
```

Comment:
That's incorrect actually, a Distribution Certificate is required for the Ad Hoc build as well (development and preview in EAS terms). 

```suggestion
The owner of the individual Apple Developer account will need to create some or all of these credentials:
- a Distribution signing certificate: needed to sign both development and release builds that are installed on a device
- Ad Hoc provisioning profile: needed to sign builds that are installed on a device outside of the App Store
- Distribution provisioning profile: needed to sign the build that is submitted to the app store
- Push key: needed to use the push notification service
```

Reply:
In our docs (and CLI) we're calling "Distribution signing certificate" "Distribution certificate" (Link: https://docs.expo.dev/app-signing/app-credentials/#distribution-certificate)?

Similarly for provisioning profile: https://docs.expo.dev/app-signing/app-credentials/#provisioning-profiles. Isn't ad-hoc word used for development only provisioning profiles?

Reply:
They're the same thing. Apple have "Provisioning Profiles" and "Signing Certificates", so "Distribution Certificate" in our docs is just a slightly abbreviated way of saying "Distribution Signing Certificate". Either is correct!

Reply:
That makes sense! Thank you for the clarification 😄